docs/docker-compose/teuthology/teuthology.sh

```diff
@@ -14,23 +14,26 @@ else
     CUSTOM_CONF=/teuthology/containerized_node.yaml
 fi
 export MACHINE_TYPE=${MACHINE_TYPE:-testnode}
-teuthology-suite -v \
-    --ceph-repo https://github.com/ceph/ceph.git \
-    --suite-repo https://github.com/ceph/ceph.git \
-    -c master \
-    -m $MACHINE_TYPE \
-    --limit 1 \
-    -n 100 \
-    --suite teuthology:no-ceph \
-    --filter-out "libcephfs,kclient,stream,centos,rhel" \
-    -d ubuntu -D 20.04 \
-    --suite-branch master \
-    --subset 9000/100000 \
-    -p 75 \
-    --seed 349 \
-    --force-priority \
-    $CUSTOM_CONF
+if [ -z "$TEUTHOLOGY_WAIT" ]; then
+    teuthology-suite -v \
+        --ceph-repo https://github.com/ceph/ceph.git \
+        --suite-repo https://github.com/ceph/ceph.git \
+        -c master \
+        -m $MACHINE_TYPE \
+        --limit 1 \
+        -n 100 \
+        --suite teuthology:no-ceph \
+        --filter-out "libcephfs,kclient,stream,centos,rhel" \
+        -d ubuntu -D 20.04 \
+        --suite-branch master \
+        --subset 9000/100000 \
+        -p 75 \
+        --seed 349 \
+        --force-priority \
+        $CUSTOM_CONF
+    DISPATCHER_EXIT_FLAG='--exit-on-empty-queue'
```

Comment:
I don't think this should be here since for the Teuthology_wait we the dispatcher to keep running even with the empty queue, but with the other mode we want to exit the empty queue.

Reply:
nvm I misread the syntax for -z and -n if statements, this looks good